cypress/e2e/RoundTrips/SimpleProject.feature

```diff
@@ -0,0 +1,473 @@
+Feature: Test home page: project creation
```

Comment:
use the same pattern as other roundtrips

Reply:
done